css/properties/grid-template-columns.json

```diff
@@ -382,6 +382,7 @@
               "firefox": [
                 {
                   "version_added": "57",
+                  "version_removed": "76",
```

Comment:
Hmm. This makes it appear as if support for `repeat()` has been dropped entirely. Unless I've misunderstood something about the bug you linked to, I think this PR is incomplete without an additional support statement above this one, with `"version_added: "76"`. I think the same goes for the other file as well.

Reply:
I've got that listed against the section with the notes, I'm not sure otherwise how to do it, do you mean to add a full support section above?

```
{
  "version_added": "76",
 },
{
  "version_added": "57",
  "version_removed": "76",
   "notes": "<code>repeat(auto-fill, ...)</code> and <code>repeat(auto-fit, ...)</code> only support one repeated column (see <a href='https://bugzil.la/1341507'>bug 1341507</a>).",
    "partial_implementation": true
 }
```

Reply:
Yep, a full object above, just like that 👍

Reply:
@ddbeck I've made it so - thanks :)